content/presubmission_inquiry.md

```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
 1. Lorem ipsum
```

Comment:
I guess this needs to be removed?

Reply:
We should probably put the rules into a separate pull request after we merge this one. It may be ideal to use some sort of template that pulls them from the `rules.md` file. Then, the rules here would be updated automatically. @agitter @dhimmel any suggestions?

Reply:
This is a presubmission inquiry right? I don't see much value to have this automatically update. It's not part of the manuscript, so I don't think our templating system would work.

Reply:
Souds good. This is probably worth committing an entire PR to. It is also slightly beyond the scope of the current PR (i.e. addressing general flow of the presubmission inquiry).